spk/python38/Makefile

```
@@ -12,7 +12,7 @@ DEPENDS += cross/lxml cross/pycrypto cross/pycurl cross/pyyaml
 DEPENDS += cross/msgpack-python cross/ruamel.yaml cross/immutables
 DEPENDS += cross/cryptography
 
-WHEELS = src/requirements.txt
+WHEELS = src/requirements-cross.txt src/requirements.txt
```

Until we have a solution for how to detect whether a wheel is pure python before we built it, we could use dedicated variables to separate the requirements files.

```suggestion
WHEELS_PURE_PYTHON = src/requirements.txt
WHEELS_CROSS_COMPILE = src/requirements-cross.txt
```

---

@th0ma7 explained this in https://github.com/SynoCommunity/spksrc/issues/4925#issuecomment-946304515